### PR TITLE
Fix configs, scripts, and demo

### PR DESCRIPTION
**Pull request by Zmoki**

### CODE REVIEW

**Comment by Zmoki** [comment]
[Widget & CKEditor: обновить](https://trello.com/c/alYu5WGU/216-widget-ckeditor-%D0%BE%D0%B1%D0%BD%D0%BE%D0%B2%D0%B8%D1%82%D1%8C)

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+**/dist/*
+**/node_modules/*
```
Comment (dmitry-mukhin) [inline .eslintignore]: no newline at EOF
change your editor settings

File: package.json [hunk 2]
```diff
@@ -2,15 +2,17 @@
   "name": "uploadcare-ckeditor",
   "version": "2.1.5",
```
Comment (dmitry-mukhin) [inline package.json]: bump version?

Comment (Zmoki) [reply package.json]: Not yet

**Pull request merged**